packages/ui/src/components/Radio/radio.tsx

```diff
@@ -0,0 +1,39 @@
+import { Radio as DefaultRadio } from 'antd'
+import React from 'react'
```

Comment:
is it new linter spring react at bottom? looks strange 😕

Reply:
> is it new linter spring react at bottom? looks strange 😕

It is)